about.html

```diff
@@ -0,0 +1,152 @@
+              <li class="menu-Item"><a href="#">Sponser</a></li>
+              <li class="menu-Item"><a href="#">News</a></li>
+              <li class="menu-Item"><a href="#">CC Campaign</a></li>
+            </ul>
+            <button class="close-btn">&times</button>
+          </div>
+        </nav>
+      </header>
+        <div class="aboutPage">
+          <div class="heading-container-about">
+           <p>"Hello! Sharing World"</p>
+           <h1 class="main-heading-about">13th ACM WEB SCIENCE CONFERENCE</h1>
+          </div>
+          <div class="about-paragraph">
+           <p>The theme for WebSci 2021, Globalisation, Inclusion and the Web in the Context of COVID, calls attention to these challenges, particularly in light of the ongoing global effects of the COVID-19 pandemic. Web Science is an interdisciplinary field dedicated to understanding the complex and multiple impacts of the Web on society, and vice versa. As such, the field of Web Science is particularly well situated to address these pressing issues.</p>
+          </div>
+          <div class="contact-text"><p>Please contact us per Email for any further questions about Web Science 2021!</p></div>
+          
+          <div class="link">
+            <p>websci21.webscience.org</p>
+          </div>
+        </div>
+      
+      </section>
+      <section class="about-icon-page">
+        <h2 class="logo-compet-title">13th ACM WEB SCIENCE CONFERENCE Logo</h2>
+        <img src="Assets/icons8-line-32.png" alt="underline" class="underline">
+        <p class="logo-text">The logo of ACM WEB SCIENCE Conference was decided through the logo competition from 15.June to 20.June.</p>
+        <div class="logo-img-about">
+            <img src="./Assets/logo.png" alt="logo-img">
+        </div>
+      </section>
+
+      <section class="past-section">
+        <h2 class="past-confers">See the past WebSci Conferences</h2>
+        <img src="Assets/icons8-line-32.png" alt="underline" class="underline">
+        <p class="past-text">Take a look at the last two WebSci confernces.</p>
+        <div class="past-photos">
+            <div class="img-past1-container">
+              <div class="img-past1-overlay">
+                <div class="img-past-texts">
+                  <h2 class="img-past-year">2020</h2>
+                  <p class="img-past-info">Making the Web Human-Centric</p>
+                </div>
+              </div>
+            </div>
+            
+            <div class="img-past2-container">
+              <div class="img-past2-overlay">
+                <div class="img-past-texts">
+                  <h2 class="img-past-year">2019</h2>
+                  <p class="img-past-info">Computer network and comunications</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        <!-- </div><img src="./Assets/websci2020.jpg"> -->
+        <!-- <img src="./Assets/websci2019.jpg"> -->
```

Comment:
[Optional] Kindly consider removing this commented code from your codebase in order to keep it clean and professional. 😉